cob-base.owl

```diff
@@ -284,17 +279,18 @@
                 <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/COB_0000011"/>
             </owl:Restriction>
         </rdfs:subClassOf>
-        <obo:IAO_0000115>A material entity that consists of two or more atoms that are all connected via covalent bonds such that any atom can be transitively connected with any other atom.</obo:IAO_0000115>
-        <obo:IAO_0000116>This molecular entity is different than ChEBI&apos;s &apos;molecular entity&apos;.</obo:IAO_0000116>
-        <obo:IAO_0000116>We would like to have cardinality restrictions on the logic, but there are some technical limitations.</obo:IAO_0000116>
-        <rdfs:label>molecule</rdfs:label>
+        <obo:IAO_0000115 xml:lang="en">A material entity that consists of two or more atoms that are all connected via covalent bonds such that any atom can be transitively connected with any other atom.</obo:IAO_0000115>
+        <obo:IAO_0000116 xml:lang="en">This molecular entity is different than ChEBI&apos;s &apos;molecular entity&apos;.</obo:IAO_0000116>
+        <obo:IAO_0000116 xml:lang="en">We would like to have cardinality restrictions on the logic, but there are some technical limitations.</obo:IAO_0000116>
+        <rdfs:label xml:lang="en">molecule</rdfs:label>
     </owl:Class>
     
 
 
     <!-- http://purl.obolibrary.org/obo/COB_0000014 -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0000014">
+        <rdfs:subClassOf rdf:resource="http://www.w3.org/2002/07/owl#Thing"/>
```

Comment:
Do we need subclass of Thing axioms in COB? Maybe use `--exclude-tautologies all` in the reason step?